templates/html-header.twig

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html {{ fn('language_attributes', 'html') }}>
+<html {{ fn('language_attributes', 'html') }} data-base="{% if constant('WP_HOME') is defined %}{{ constant('WP_HOME') }}{% endif %}">
```

Comment:
I added this because the theme editor uses local storage to store the editor state. It checks for the `data-base` attribute to create a local storage namespace. All sites on the same domain share the local storage, so if we don't provide a namespace these will unintentionally be shared. See https://github.com/Inwerpsel/use-theme-editor/blob/main/src/useLocalStorage.js#L3-L5